src/components/Wizard/SpotlightWizard/messageTypes.ts

```diff
@@ -16,9 +18,43 @@ export default interface SpotlightMessageContent {
 export interface SpotlightScreen {
   id: string;
   content: Record<string, unknown>;
+
+  [key: string]: unknown;
 }
 
 export interface SpotlightAction {
   navigate?: true;
   dismiss?: true;
 }
+
+export interface LogoAndTitleScreenContent {
+  logo: SpotlightLogo;
+  background?: string;
+  title: LocalizableText;
+  title_style?: "fancy" | "fancy shine";
+  subtitle?: LocalizableText;
+  primary_button: SpotlightButton;
+  secondary_button?: SpotlightButton;
+  dismiss_button?: SpotlightDismissButton;
+
+  [key: string]: unknown;
+}
+
+export interface SpotlightLogo {
+  imageURL?: string;
+  height?: string;
+}
+
+export interface SpotlightButton {
+  label: LocalizableText;
+  action: SpotlightAction & {
+    type?: string;
+    data?: Record<string, unknown>;
```

Comment:
Looks like data is still showing up as required, which is making actions without it fail